src/index.ts

```diff
@@ -250,6 +250,10 @@ const isFile = (path?: string | undefined): path is string => {
   }
 }
 
+const isModule = (path?: string | undefined): path is string => {
+  return !!path && isFile(`${path}/package.json`)
```

Comment:
It'll fail on Windows.

Reply:
Changed this to use `path.sep` instead of `/`